src/scirpy/pl/_logoplots.py

```diff
@@ -0,0 +1,177 @@
+from collections.abc import Sequence
+from typing import Literal, Union
+
+import palmotif as palm
+import pandas as pd
+from IPython.display import SVG
+
+from scirpy.get import airr as get_airr
+from scirpy.util import DataHandler
+
+
+@DataHandler.inject_param_docs()
+def logoplot_cdr3_motif(
```

Comment:
please add the function to the API documentation: https://github.com/scverse/scirpy/blob/main/docs/api.rst